pkg/services/chgm/chgm.go

```diff
@@ -95,7 +95,7 @@ func isUserAllowedToStop(username, issuerUsername string, userDetails CloudTrail
 	for _, operatorIamName := range operatorIamNames {
 		// I added a '-' in the end to make sure it's the same as close as I can to the aws username (cannot guess the uuid)
 		operatorIamName = fmt.Sprintf("%s-%s-", infraID, operatorIamName)
-		if strings.HasPrefix(username, operatorIamName) {
+		if strings.Contains(username, operatorIamName) {
```

Comment:
**Question:** Why is this change necessary? Did the formatting for the username change, or did this never work to begin with?

Reply:
some STS role names do not conform to the infraID-operatorIamName format, so we were not properly escalating some CHGM before